gluoncv/utils/__init__.py

```diff
@@ -10,6 +10,7 @@
 from .download import download, check_sha1
 from .filesystem import makedirs
 from .filesystem import try_import_dali
+from .filesystem import try_import_cv2
```

Comment:
can we have all `filesystem` functions imported in the same lin (which i believe is perferred by pylint)?

Reply:
Fixed